OPERATOR.rst

```diff
@@ -448,6 +448,45 @@ not, then the PR's original author should.
 Once the base branch is restored, the ``Reopen PR`` button should again be
 clickable on the chained PR.
 
+Integration test time out
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This can happen in the rare occasion that the IT's random selection of bundles
+happens to pick predominantly large bundles that need to be divided before they
+can be indexed. In the AWS console, run the CloudWatch query below with the
+indexer log groups selected to see how many divisions have occurred::
```

Comment:
Add note that not only bundles can be divided into partitions, but also partitions into sub-partitions. Technically, bundles are partitions with an empty prefix.